geode-protobuf/src/test/java/org/apache/geode/protocol/RoundTripCacheConnectionJUnitTest.java

```diff
@@ -125,13 +128,17 @@ public void setup() throws Exception {
     cacheFactory.set(ConfigurationProperties.MCAST_PORT, "0");
     cacheFactory.set(ConfigurationProperties.ENABLE_CLUSTER_CONFIGURATION, "false");
     cacheFactory.set(ConfigurationProperties.USE_CLUSTER_CONFIGURATION, "false");
+
+
     cache = cacheFactory.create();
 
     CacheServer cacheServer = cache.addCacheServer();
     cacheServerPort = AvailablePortHelper.getRandomAvailableTCPPort();
     cacheServer.setPort(cacheServerPort);
+    cacheServer.setMaximumTimeBetweenPings(200);
```

Comment:
Why would we change the MaximumTimeBetweenPings value for all tests? Would it not make sense to specifically start a cacheServer with the required values and test against that cacheServer, rather than change the behavior for all?